### PR TITLE
Add /fixlight chat command

### PR DESCRIPTION
**Pull request by juhdanad**
Addresses https://github.com/minetest/minetest/pull/5373#issuecomment-295544082.

Syntax: `/fixlight (here <radius>) | (<pos1> <pos2>)`, same as `\deleteblocks` has.
Required privilege: server

### CODE REVIEW

File: builtin/game/chatcommands.lua [hunk 1]
```diff
@@ -487,6 +487,26 @@ core.register_chatcommand("deleteblocks", {
 	end,
 })
 
+core.register_chatcommand("fixlight", {
+	params = "(here [radius]) | (<pos1> <pos2>)",
+	description = "Removes lighting bugs in the area between pos1 and pos2 "
+		.. "(light might actually change in a larger volume)",
+	privs = {server=true},
```
Comment (paramat) [inline builtin/game/chatcommands.lua]: Spaces around '=' (could be fixed on merge).

File: builtin/game/chatcommands.lua [hunk 2]
```diff
@@ -487,6 +487,26 @@ core.register_chatcommand("deleteblocks", {
 	end,
 })
 
+core.register_chatcommand("fixlight", {
+	params = "(here [radius]) | (<pos1> <pos2>)",
+	description = "Removes lighting bugs in the area between pos1 and pos2 "
+		.. "(light might actually change in a larger volume)",
```
Comment (paramat) [inline builtin/game/chatcommands.lua]: I don't think this line is needed, it isn't present in other similar commands, the warning can be in documentation instead.

File: builtin/game/chatcommands.lua [hunk 3]
```diff
@@ -487,6 +487,26 @@ core.register_chatcommand("deleteblocks", {
 	end,
 })
 
+core.register_chatcommand("fixlight", {
+	params = "(here [radius]) | (<pos1> <pos2>)",
+	description = "Removes lighting bugs in the area between pos1 and pos2 "
```
Comment (paramat) [inline builtin/game/chatcommands.lua]: Perhaps better to not mention lighting bugs, instead more precise to say 'updates lighting ..' or something.

File: builtin/game/chatcommands.lua [hunk 4]
```diff
@@ -487,6 +487,26 @@ core.register_chatcommand("deleteblocks", {
...
+		.. "(light might actually change in a larger volume)",
+	privs = {server=true},
+	func = function(name, param)
+		local p1, p2 = parse_range_str(name, param)
+		if p1 == false then
+			return false, p2
+		end
+
+		if core.fix_light(p1, p2) then
+			return true, "Successfully corrected light in the area ranging from " ..
```
Comment (paramat) [inline builtin/game/chatcommands.lua]: 'Succesfully updated light ..'

**Comment by paramat** [comment]
+1 once line comments addressed and if tested (it probably is).

**Comment by juhdanad** [comment]
@paramat Done!

**Comment by paramat** [comment]
:+1:

**Comment by paramat** [comment]
f6d1b682d3c3fa25e11d64d52c93b92760fe2612

**Pull request closed without merging**